### PR TITLE
docs(tabs): Page objects to jsdoc3

### PR DESCRIPTION
**Pull request by Droogans**
### LGTMS
- [x] Dev LGTM
- [x] QE LGTM

### CODE REVIEW

File: src/components/tabs/scripts/tabs.page.js [hunk 1]
```diff
@@ -21,6 +54,16 @@ var tabFromElement = function (tabElement) {
...
         },
 
+        /**
+         * @instance
+         * @type {String}
+         * @memberof tabs.tab
+         * @description The name of the tab. Will parse out the subtitle, if it exists.
+         * @example
+         * it('should have just the name correct', function () {
+         *     expect(encore.tabs.initialize().byName('Activity').name).to.eventually.equal('Activity');
```
Comment (CITguy) [inline src/components/tabs/scripts/tabs.page.js]: Can you use a `var tab = ...` like the example in `fullName`, to reduce example line length, and improve readability?

File: src/components/tabs/scripts/tabs.page.js [hunk 2]
```diff
@@ -38,6 +81,16 @@ var tabFromElement = function (tabElement) {
...
         },
 
+        /**
+         * @instance
+         * @type {String}
+         * @memberof tabs.tab
+         * @description The subtitle of the tab. Will parse out the name, if it exists.
+         * @example
+         * it('should have just the subtitle correct', function () {
+         *     expect(encore.tabs.initialize().byName('Activity').subtitle).to.eventually.equal('recent first');
```
Comment (CITguy) [inline src/components/tabs/scripts/tabs.page.js]: Same comment wrt `var tab = ...` above.

**Comment by CITguy** [comment]
LGTM

**Comment by jj4th** [comment]
QE-LGTM

**Pull request merged**